sui-execution/src/lib.rs

```diff
@@ -15,20 +15,37 @@ pub mod verifier;
 mod latest;
 mod v0;
 
+pub const MIN_EXECUTOR_VERSION: u64 = 0;
+pub const LATEST_EXECUTOR_VERSION: u64 = 1;
```

Comment:
Is this part of the PR necessary?  It complicates automating the creation of new versions of the executor interface.